src/views/index.ejs

```diff
@@ -1,9 +1,5 @@
 <!DOCTYPE html>
```

Comment:
This markup is unnecessary for this since the response is anyway gonna be added directly to the dom